README.md

```diff
@@ -3,12 +3,73 @@ o-card [![Circle CI](https://circleci.com/gh/Financial-Times/o-card/tree/master.
 
 Origami component for a card.
 
-## Types of Card
+## Browser support
 
-A card can be one of 1 type
-- Standard `.o-card`
+|  Browsers  | Primary Experience | Core Experience |
+|:----------:|:------------------:|:---------------:|
+|   Chrome   |        35+         |       35+       |
+|   Firefox  |        30+         |       30+       |
+|   Safari   |        7+          |       7+        |
+|   IE       |        8+          |       8+        |
+
+Known issues:
+
+* Image alignment is not supported in <IE9, the image will default to being full width of the card and aligned above the timestamp
```

Comment:
That was the case, I'm working on it now though to make sure the alignment works for all browsers that get enhanced support.